src/NugetFeedDeploymentRepository/NugetFeedCredentials.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Etg.Yams.NuGet.Storage
+{
+    public class NugetFeedCredentials
+    {
+        public NugetFeedCredentials(string username, string password)
+        {
+            this.Username = username;
+            this.Password = password;
+        }
+
+        public string Username { get; private set; }
```

Comment:
I noticed you always add private set, what for? I would avoid them unless needed.